components/VocabularyOverviewListItem/index.tsx

```diff
@@ -18,15 +19,21 @@ const VocabularyOverviewListItem = ({
   image,
   article,
   word,
+  audio,
 }: IVocabularyOverviewListItemProps) => {
-  const handlaSpeakerClick = () => {
-    Tts.speak(word, {
-      androidParams: {
-        KEY_PARAM_PAN: -1,
-        KEY_PARAM_VOLUME: 0.5,
-        KEY_PARAM_STREAM: 'STREAM_MUSIC',
-      },
-    });
+  const handlaSpeakerClick = (audio: string) => {
+    if (audio) {
+      //audio from API
+      SoundPlayer.playUrl(audio);
```

Comment:
Something seems to be wrong with which audio default. The default isnt using TTS and doesn't work on IOS